index.js

```diff
@@ -31,13 +31,18 @@ var COLORS = {
   }
 }
 
-function TorrentGraph (root) {
+function TorrentGraph (root, size) {
   if (typeof root === 'string') root = document.querySelector(root)
   var model = {
     nodes: [],
     links: []
   }
 
+  if (size) {
+    if (size.height && typeof size.height !== 'function' && size.height !== 'default' && size.height !== 'auto' && typeof size.height !== 'number') throw new Error('Height should be a Number or a Function')
+    if (size.width && size.width !== 'default' && typeof size.width !== 'function' && size.width !== 'auto' && typeof size.width !== 'number') throw new Error('Width should be a Number of a Function')
```

Comment:
Long lines are hard to read. It's usually best to hit enter so they don't get this long.